### DISCUSSION TITLE
Batteria - template

### DISCUSSION THREAD

**Discussion by sunsetitaly** [indomus/forum#1734] [HUB: Home Assistant]
Per verificare la percentuale di carica della batteria di un sensore TH, vorrei salvare il valore iniziale di essa (perché nei diversi cicli di carica ho verificato che non è sempre uguale) per poter calcolare la percentuale del valore attuale rispetto a quello iniziale.
Non saprei come salvare il valore iniziale della batteria ad una variable, che ritenga il valore anche di fronte ad un reset del sensore.
Riuscite a darmi qualche indicazione? grazie in anticipo

**Comment by indomus** [comment]
Non ti impelagare:

- ogni batteria ha un valore iniziale diverso;
- le batterie vengono lette in modo variabile anche in funzione della temperatura e della variabilità nell'assorbimento.

Basati piuttosto su una soglia arbitraria sul valore intero che hai.

**Reply by sunsetitaly** [reply]
Grazie, mi sa che così lo farò. Fisserò 4,2 per Ion-Li e 3,4V per LiFePo4
